content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -36,12 +36,14 @@ IPv4/IPv6 dual-stack on your Kubernetes cluster provides the following features:
 
 ## Prerequisites
 
-The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack Kubernetes clusters:
+A version of Kubernetes that supports dual-stack networking. Dual-stack networking is generally available
+since Kubernetes v1.23, so if you are running Kubernetes v{{< skew currentVersion >}} then this
+feature is available to you.
+If you are not running Kubernetes v{{< skew currentVersion >}}, check the documentation for
+that version of Kubernetes. {{< version-check >}}
+
+Your cluster must also meet the following requirements:
 
-   * Kubernetes 1.20 or later  
-     For information about using dual-stack services with earlier
-     Kubernetes versions, refer to the documentation for that version
-     of Kubernetes.
```

Comment:
Imo, we should keep this section.
Dual-stack implementation is very different prior to 1.20
From 1.15 to 1.20, "dual-stack" implementation required a Service per IP family

Reply:
We could reword it. If we direct people to the docs for older Kubernetes, it behooves us to serve those docs (we currently don't, although _aside_ I believe @nate-double-u has a cunning plan there).

So, let's think about how to help people but not set a false expectation.

Reply:
@reylejano @sftim Apart from your conversation this PR looks inactive, so I'm going to `/close` it. Please feel free to continue the discussion and possibly revive the PR.